index.js

```diff
@@ -15,3 +15,128 @@ Array.from(closes).forEach((close) => {
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop-2.png',
+  },
+  {
+    project_id: 'project3',
+    name: 'Facebook 360',
+    experiance: ['Facebook', 'Backend Dev', '2015'],
+    description:
+      'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop3.jpg',
+  },
+  {
+    project_id: 'project4',
+    name: 'Uber Navigation',
+    experiance: ['Uber', 'Backend Dev', '2015'],
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum',
+    image: 'images/Snapshoot-Porfolio-desktop4.png',
+  },
+];
+
+const openModalButtons = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+
+function openModal(modal) {
+  if (modal == null) return;
+
+  const modals = document.querySelectorAll('.modal');
+
+  for (let i = 0; i <= projects.length; i += 1) {
+    if (projects[i].project_id === modal.id) {
+      modals.forEach((modal) => {
+        const htmlBody = document.querySelector('body');
+        const title = modal.firstElementChild.firstElementChild;
+        const body = modal.lastElementChild;
+        htmlBody.style.overflow = 'hidden';
+        title.innerHTML = `<h3> ${projects[i].name} </h3>`;
+        body.innerHTML = `<ul id = "experience"> ${projects[i].experiance[0]}
+                            <li class = "type"> ${projects[i].experiance[1]} </li> 
+                            <li class = "type"> ${projects[i].experiance[2]} </li>
+                          </ul>
+                          <img class = "modal-img" src="images/snapshoot-portfolio-3.svg" alt="">
```

Comment:
- [ ] Please instead of making the image fixed in every project pop-up window use the image in project object example `<img class = "modal-img" src="${projects[i].image}" alt="">`.